powerpoint.gemspec

```diff
@@ -4,24 +4,24 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'powerpoint/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "powerpoint"
+  spec.name          = 'powerpoint'
```

Comment:
Turns out that in Ruby 2.0 the interpolation time really doesn't account for much.